### PR TITLE
v3.3.2

### PR DESCRIPTION
**Pull request by milesmanley**
Mitigates the reported security vuln reported [here](https://z.cash/support/security/announcements/security-announcement-2019-09-24/) as well as as a version bump.

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -14,6 +20,12 @@ Miles Manley (3)
 Jules "Coinblockers" (1)
         Fix untar ownership fail in containerized environments
 
+Changlog - Zcash 2.0.6
```
Comment (TheTrunk) [inline doc/release-notes.md]: this is 2.0.7-3

Comment (milesmanley) [reply doc/release-notes.md]: I think you think this is a different pr?

**Comment by milesmanley** [comment]
> Furthermore versioning is not correctly set

I might be blind to this? Where is versioning not set correctly? - might be something obvious.

**Comment by TheTrunk** [comment]
configure.ac :D

**Comment by milesmanley** [comment]
Ok - @TheTrunk @Lumiboy please review and approve - so we can merge and release - In the mean time I will see if I can fix the Sprout to Sapling Migration tool quickly - so we can include that - but will do it in a separate branch to keep things clean.

**Pull request merged**